README.rst

```diff
@@ -117,6 +117,17 @@ These are both equivalent to running::
 
     codespell --quiet-level 3 --count --skip "*.po,*.ts,./src/3rdParty,./src/Test"
 
+If several config files are present, they are read in the following order:
+
+#. ``pyproject.toml``
```

Comment:
Only if tomli or tomllib are present?

Reply:
good catch, only if `tomli` is installed (tomllib is only python>3.11 and not conditionally imported in this module as far as I can see). I'll push a commit tomorrow.

Reply:
https://twitter.com/gvanrossum/status/1583561788204806144?s=20&t=-A2jdKe7r-ZgRUtSA8JZ_g

Reply:
documented in https://github.com/codespell-project/codespell/pull/2552/commits/2381c2b100a0d3dfde0455334228d5256f488244 -- adding `tomllib` would be more involved and should be done in a separate, dedicated PR.